### ISSUE TITLE
Heroku Banned

### ISSUE THREAD

**Issue by kamileecher** [wy580477/qBit-to-Rclone-on-Heroku#3]

**Comment by wy580477** [comment]
Well, I just created a new heroku accout to test deployment. It worked.

Did your app get banned after a while?

**Comment by kamileecher** [comment]
yes not banned immediately.  It can not ban a few hours later.  This is our banlaniy and other applications.

**Comment by wy580477** [comment]
> yes not banned immediately. It can not ban a few hours later. This is our banlaniy and other applications.

Well, if you got banned after downloading files I think perhaps heroku is banning dynos based on traffic analysis or dns requests.

I don't have solution for this.

I'm going to archive this repository in a few days.

**Comment by wy580477** [comment]
> yes not banned immediately. It can not ban a few hours later. This is our banlaniy and other applications.

Did a little trick in code, I think it can evade Heroku's detection for now.
